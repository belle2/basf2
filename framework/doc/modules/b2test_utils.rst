.. automodule:: b2test_utils
   :members:
