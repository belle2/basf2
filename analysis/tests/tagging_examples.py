#!/usr/bin/env python3
# -- analysis/tests/tagging_examples.py

##########################################################################
# basf2 (Belle II Analysis Software Framework)                           #
# Author: The Belle II Collaboration                                     #
#                                                                        #
# See git log for contributors and copyright holders.                    #
# This file is licensed under LGPL-3.0, see LICENSE.md.                  #
##########################################################################

import b2test_utils as b2tu
import b2test_utils_analysis as b2tua

#: extend the examples test class to actually run over the tagging examples:


class ExamplesTest(b2tua.ExamplesTest):
    def test_tagging_examples(self):
        """
        Test supported tagging examples.
        """

        self._test_examples_dir('analysis/examples/tagging/')


if __name__ == '__main__':
    with b2tu.clean_working_directory():
        b2tua.unittest.main()
