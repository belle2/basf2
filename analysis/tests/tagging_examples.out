[INFO] Steering file: analysis/tests/tagging_examples.py
TFile: BtagBsigReconstruction.root
TTree: btagbsig
TBranch: __experiment__, 0
TBranch: __run__, 0
TBranch: __event__, 3
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 37
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: isSignal, nan
TBranch: mcErrors, 512
TBranch: mcPDG, nan
TBranch: nROE_Charged__bo__bc, 0
TBranch: nROE_Photons__bo__bc, 8
TBranch: nROE_NeutralHadrons__bo__bc, 4
TBranch: pRecoil, 2.485
TBranch: eRecoil, 1.742
TBranch: mRecoil, -1.773
TBranch: pRecoilPhi, 0.5754
TBranch: pRecoilTheta, 1.936
TBranch: pxRecoil, 1.948
TBranch: pyRecoil, 1.263
TBranch: pzRecoil, -0.8881
TBranch: roeEextra__bo__bc, 4.12
TBranch: dmID, 1
TBranch: Btag_isSignal, 1
TBranch: Btag_mcErrors, 0
TBranch: Btag_mcPDG, -511
TBranch: Btag_Mbc, 5.282
TBranch: Btag_deltaE, -0.052
TBranch: Bsig_isSignal, nan
TBranch: Bsig_mcErrors, 512
TBranch: Bsig_mcPDG, nan
TBranch: Bsig_Mbc, 4.585
TBranch: Bsig_deltaE, -1.927
TTree: btag
TBranch: __experiment__, 0
TBranch: __run__, 0
TBranch: __event__, 3
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 1
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: isSignal, 1
TBranch: mcErrors, 0
TBranch: mcPDG, -511
TBranch: Mbc, 5.282
TBranch: deltaE, -0.052
TBranch: nROE_Charged__bo__bc, 2
TBranch: nROE_Photons__bo__bc, 10
TBranch: nROE_NeutralHadrons__bo__bc, 4
TFile: KlongDecayReconstructionExample.root
TTree: tree
TBranch: __experiment__, 0
TBranch: __run__, 0
TBranch: __event__, 2
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 4
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: M, 5.28
TBranch: ErrM, 0.005402
TBranch: SigM, 0
TBranch: InvM, 5.278
TBranch: isSignal, 0
TBranch: mcErrors, 384
TBranch: mcPDG, 443
TBranch: px, 4.901
TBranch: py, -1.713
TBranch: pz, -0.1299
TBranch: pt, 5.191
TBranch: p, 5.193
TBranch: E, 7.405
TBranch: mcE, 3.441
TBranch: mcP, 1.499
TBranch: mcPT, 1.494
TBranch: mcPX, 1.03
TBranch: mcPY, -1.082
TBranch: mcPZ, 0.1214
TBranch: mcPhi, -0.8104
TBranch: distance, 0.03273
TBranch: significanceOfDistance, 1.873
TBranch: dx, 0.004896
TBranch: dy, 0.001033
TBranch: dz, 0.03235
TBranch: x, 0.004911
TBranch: y, 0.001112
TBranch: z, 0.03164
TBranch: x_uncertainty, 0.001485
TBranch: y_uncertainty, 0.0008241
TBranch: z_uncertainty, 0.001045
TBranch: dr, 0.005004
TBranch: dphi, 0.2079
TBranch: dcosTheta, 0.9882
TBranch: prodVertexX, nan
TBranch: prodVertexY, nan
TBranch: prodVertexZ, nan
TBranch: prodVertexXErr, nan
TBranch: prodVertexYErr, nan
TBranch: prodVertexZErr, nan
TBranch: chiProb, 0.9456
TBranch: mcDecayVertexX, 0.001414
TBranch: mcDecayVertexY, 0.0006351
TBranch: mcDecayVertexZ, 0.03091
TBranch: mcDecayVertexFromIPDistance, 0.03165
TBranch: mcDecayVertexRho, 0.00155
TBranch: mcProductionVertexX, 0.001414
TBranch: mcProductionVertexY, 0.0006351
TBranch: mcProductionVertexZ, 0.03091
TBranch: Mbc, nan
TBranch: deltaE, 2.24
TBranch: DeltaT, -0.3198
TBranch: DeltaTErr, 0.2311
TBranch: DeltaZ, -0.00352
TBranch: DeltaZErr, 0.002147
TBranch: DeltaBoost, -0.002749
TBranch: DeltaBoostErr, 0.001986
TBranch: TagVLBoost, 0.03477
TBranch: TagVLBoostErr, 0.001696
TBranch: TagVOBoost, -0.005235
TBranch: TagVOBoostErr, 0.002092
TBranch: TagVpVal, 0.0002583
TBranch: TagVNDF, 9
TBranch: TagVChi2, 31.34
TBranch: TagVChi2IP, 305.5
TBranch: TagVx, 5.795e-05
TBranch: TagVxErr, 0.001932
TBranch: TagVy, 0.001622
TBranch: TagVyErr, 0.0009426
TBranch: TagVz, 0.03516
TBranch: TagVzErr, 0.001876
TBranch: mcDeltaTau, nan
TBranch: mcDeltaT, nan
TBranch: mcDeltaBoost, nan
TBranch: mcTagVLBoost, nan
TBranch: mcTagVOBoost, nan
TBranch: mcLBoost, 0.03077
TBranch: mcOBoost, -0.003254
TBranch: mcTagVx, nan
TBranch: mcTagVy, nan
TBranch: mcTagVz, nan
TBranch: nTracks, 9
TBranch: Jpsi_M, 3.094
TBranch: Jpsi_ErrM, 0
TBranch: Jpsi_SigM, -inf
TBranch: Jpsi_InvM, 3.094
TBranch: Jpsi_isSignal, 1
TBranch: Jpsi_mcErrors, 0
TBranch: Jpsi_mcPDG, 443
TBranch: Jpsi_px, 1.023
TBranch: Jpsi_py, -1.084
TBranch: Jpsi_pz, 0.1231
TBranch: Jpsi_pt, 1.49
TBranch: Jpsi_p, 1.495
TBranch: Jpsi_E, 3.436
TBranch: Jpsi_mcE, 3.441
TBranch: Jpsi_mcP, 1.499
TBranch: Jpsi_mcPT, 1.494
TBranch: Jpsi_mcPX, 1.03
TBranch: Jpsi_mcPY, -1.082
TBranch: Jpsi_mcPZ, 0.1214
TBranch: Jpsi_mcPhi, -0.8104
TBranch: Jpsi_distance, 0.0007083
TBranch: Jpsi_significanceOfDistance, 0.04039
TBranch: Jpsi_dx, -1.471e-05
TBranch: Jpsi_dy, -7.917e-05
TBranch: Jpsi_dz, 0.0007037
TBranch: Jpsi_x, 0
TBranch: Jpsi_y, 0
TBranch: Jpsi_z, 0
TBranch: Jpsi_x_uncertainty, 0
TBranch: Jpsi_y_uncertainty, 0
TBranch: Jpsi_z_uncertainty, 0
TBranch: Jpsi_dr, 8.052e-05
TBranch: Jpsi_dphi, -1.754
TBranch: Jpsi_dcosTheta, 0.9935
TBranch: Jpsi_prodVertexX, nan
TBranch: Jpsi_prodVertexY, nan
TBranch: Jpsi_prodVertexZ, nan
TBranch: Jpsi_prodVertexXErr, nan
TBranch: Jpsi_prodVertexYErr, nan
TBranch: Jpsi_prodVertexZErr, nan
TBranch: Jpsi_chiProb, -1
TBranch: Jpsi_mcDecayVertexX, 0.001414
TBranch: Jpsi_mcDecayVertexY, 0.0006351
TBranch: Jpsi_mcDecayVertexZ, 0.03091
TBranch: Jpsi_mcDecayVertexFromIPDistance, 0.03165
TBranch: Jpsi_mcDecayVertexRho, 0.00155
TBranch: Jpsi_mcProductionVertexX, 0.001414
TBranch: Jpsi_mcProductionVertexY, 0.0006351
TBranch: Jpsi_mcProductionVertexZ, 0.03091
TBranch: KL0_M, 0.4976
TBranch: KL0_ErrM, 0
TBranch: KL0_SigM, inf
TBranch: KL0_InvM, 0.4976
TBranch: KL0_isSignal, 0
TBranch: KL0_mcErrors, 128
TBranch: KL0_mcPDG, 13
TBranch: KL0_px, 3.878
TBranch: KL0_py, -0.6293
TBranch: KL0_pz, -0.253
TBranch: KL0_pt, 3.929
TBranch: KL0_p, 3.937
TBranch: KL0_E, 3.968
TBranch: KL0_mcE, 2.276
TBranch: KL0_mcP, 2.273
TBranch: KL0_mcPT, 2.27
TBranch: KL0_mcPX, 2.126
TBranch: KL0_mcPY, -0.7958
TBranch: KL0_mcPZ, -0.1274
TBranch: KL0_mcPhi, -0.3582
TBranch: KL0_distance, 0.0007083
TBranch: KL0_significanceOfDistance, 0.04039
TBranch: KL0_dx, -1.471e-05
TBranch: KL0_dy, -7.917e-05
TBranch: KL0_dz, 0.0007037
TBranch: KL0_x, 0
TBranch: KL0_y, 0
TBranch: KL0_z, 0
TBranch: KL0_x_uncertainty, 0
TBranch: KL0_y_uncertainty, 0
TBranch: KL0_z_uncertainty, 0
TBranch: KL0_dr, 8.052e-05
TBranch: KL0_dphi, -1.754
TBranch: KL0_dcosTheta, 0.9935
TBranch: KL0_prodVertexX, nan
TBranch: KL0_prodVertexY, nan
TBranch: KL0_prodVertexZ, nan
TBranch: KL0_prodVertexXErr, nan
TBranch: KL0_prodVertexYErr, nan
TBranch: KL0_prodVertexZErr, nan
TBranch: KL0_chiProb, -1
TBranch: KL0_mcDecayVertexX, 800
TBranch: KL0_mcDecayVertexY, -129.1
TBranch: KL0_mcDecayVertexZ, -74.3
TBranch: KL0_mcDecayVertexFromIPDistance, 813.8
TBranch: KL0_mcDecayVertexRho, 810.4
TBranch: KL0_mcProductionVertexX, 0.001414
TBranch: KL0_mcProductionVertexY, 0.0006351
TBranch: KL0_mcProductionVertexZ, 0.03091
TBranch: CMS_px, 4.597
TBranch: CMS_py, -1.713
TBranch: CMS_pz, -2.179
TBranch: CMS_pt, 4.906
TBranch: CMS_p, 5.368
TBranch: CMS_E, 7.529
TBranch: Jpsi_CMS_px, 0.8809
TBranch: Jpsi_CMS_py, -1.084
TBranch: Jpsi_CMS_pz, -0.8351
TBranch: Jpsi_CMS_pt, 1.396
TBranch: Jpsi_CMS_p, 1.627
TBranch: Jpsi_CMS_E, 3.496
TBranch: KL0_CMS_px, 3.716
TBranch: KL0_CMS_py, -0.6293
TBranch: KL0_CMS_pz, -1.343
TBranch: KL0_CMS_pt, 3.769
TBranch: KL0_CMS_p, 4.001
TBranch: KL0_CMS_E, 4.032
TBranch: dmID, 0
